### ISSUE TITLE
Create and maintain changelogs

### ISSUE THREAD

**Issue by t-mullen** [multihack/multihack-web#41]
I should've done this from the start, but proper changelogs and releases will start by/before Beta release.

**Comment by tweakimp** [comment]
Do we use semantic versioning?
Will we write an *.md file or use something else?

**Comment by t-mullen** [comment]
Yes, we already use semver. I'd prefer not to make a new release every patch like I am now though. This only applies to the extensions, the web version can continue being continuous delivery.

Yeah, just a "changelog.md" file. I haven't bothered maintaining one out of no excuse but laziness. When the Beta comes out, I'll start.

**Comment by tweakimp** [comment]
Ok, I will create a template for that to make it pretty ;)
